translator/translate/otel/processor/jmxprocessor/translator.go

```diff
@@ -0,0 +1,156 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package jmxprocessor
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/processor"
+
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/common"
+)
+
+const (
+	regexp = "regexp"
+)
+
+var (
+	jmxKey     = common.ConfigKey(common.MetricsKey, common.MetricsCollectedKey, common.JmxKey)
+	configKeys = map[component.DataType]string{
+		component.DataTypeMetrics: common.ConfigKey(jmxKey),
+	}
```

Comment:
This map isn't necessary. The JMX filter isn't going to be used with non-metric pipelines.